src/structs.jl

```diff
@@ -247,6 +247,17 @@ end
 
 """
 Data structure used to store information and results for a stochastic programming problem.
+
+*`r::AbstractPenaltyParameter`
+*`scenario_tree::ScenarioTree`
+*`scenario_map::Dict{ScenarioID, ScenarioInfo}`
+*`callbacks::Vector{Callback}`
+*`xhat::Dict{XhatID, HatVariable}`
+*`variable_data::Dict{VariableID, VariableData}`
+*`indexer::Indexer`
+*`iterate_history::PHIterateHistory`
+*`residual_history::PHResidualHistory`
+*`time_info::TimerOutputs.TimerOutput`
```

Comment:
This does need some documentation.  I think a better way to go is to refer to the available functions for accessing things.  Something like
```
...
For interacting with this object see: [`consensus_variables`](@ref)
...
```

The idea is that users shouldn't (for the most part) need to reach into the internals of the `PHData` object but interact with it through functions.  That way we do not expose internals that the user doesn't need and could break.

Reply:
Also the two built in Julia functions `methods` and `methodswith` are extremely helpful in these situations.